inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -683,9 +683,6 @@
                     "id": {
                         "type": "string"
                     },
-                    "set": {
```

Comment:
Could be multiple. Actually I wonder if this has to be ported at all... This is used by OAI-PMH server to decide which records to expose for which set. I think this is going to be overhauled by `inveniosoftware/invenio-oaiserver`. 

What about dropping the `oai_pmh` field entirely everywhere?